po/fr.po

```diff
@@ -2017,7 +2012,6 @@ msgid "<filename containing invalid UTF-8 codepoint>"
 msgstr "<fichier contenant un code UTF-8 invalide>"
 
 #: rust/libnewsboat/src/fslock.rs:77
-#, fuzzy
 msgid "Failed to write PID to lock file '%s': %s"
 msgstr "Echec d'écriture du PID dans le fichier de verrouillage : '%s' (%s)"
```

Comment:
Please check this translation, as the punctuation at the end differs between original and translation.